tests/metrics-port/00-assert.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: metrics-port
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metrics-port
+spec:
+  selector:
+    matchLabels:
+      app: metrics-port
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+      labels:
+        app: metrics-port
+    spec:
+      containers:
+        - name: metrics-port
+          image: image
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+            - containerPort: 8181
+              protocol: TCP
```

Comment:
Could probably be achieved, yes. Will take a closer look at how Skiperator is set up to see if this can be done without breaking changes, but gut feeling points to "yes".

Reply:
The field to expose the "main" port is only implemented as an integer in the application.yaml, and adding the ability to name and define protocol for it would likely result in a breaking change. But for the additional ports I have implemented them as a list of Port objects, with the ability to specify name and protocol in conjunction with the port numbers.

Reply:
I see. My initial thought that just to name the port in skiperator and not exposing the possibility to give a custom name in the manifest (and thus keeping backward compatibility).